Gruntfile.js

```diff
@@ -104,11 +103,22 @@ module.exports = function(grunt) {
         branch: 'master'
       },
       src: ['**']
-    }
+    },
+    smoosher: {
+      options: {
+        jsDir: "_site/",
+        cssDir: "_site/"
+      },
+      all: {
+        files: {
+          '_site/index.html': '_site/index.html',
+        },
+      },
+    },
   });
 
-  grunt.registerTask('build', ['less','jekyll:build']);
-  grunt.registerTask('optimize', ['imagemin','cssmin','uglify','htmlmin']);
+  grunt.registerTask('build', ['less','jekyll:build','uncss','cssmin']);
```

Comment:
Sort of, but the min.css file is referenced by the html so it needs to be built.